### PR TITLE
Example Models SSS 3 and SSS 4

### PR DESCRIPTION
**Pull request by elizabethcbailey**

### CODE REVIEW

File: validation/SSS 3/model.json [hunk 1]
```diff
@@ -0,0 +1,361 @@
+{
+    "id": "435a14a8-50bb-4ae0-9686-c8c05a37a073",
+    "metadata": {
+        "title": "Dam with a concret cutoff - RPS Feb23",
+        "description": "test",
+        "jobNumber": "00003",
+        "owner": "e94fedd3-c151-5d43-ad8b-3d78cb48bbce",
```
Comment (Alvaro-Rodrigueez) [inline validation/SSS 3/model.json:7]: Could you change the owner to be "710fe8da-8cb3-41c1-807b-13e20e0a59d1" that's the special owner for all the example models

**Review by Alvaro-Rodrigueez — CHANGES_REQUESTED** [review]
SSS 4 only has the image, all other files are empty

**Review by Alvaro-Rodrigueez — APPROVED** [review]

**Comment by elizabethcbailey** [comment]
image for SSS 4 deleted.

File: validation/SSS 3/model.json (hunk 1 above)
Comment (elizabethcbailey) [reply validation/SSS 3/model.json]: Owner udpated.

**Pull request merged**